### PR TITLE
Pioneer 18 Glider Deployments

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Deployment configurations can be reviewed from glider FCPs: 
339: 3304-00150-00153
380: 3304-00150-00154
388: 3304-00150-00155

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 1]
```diff
@@ -82,4 +82,10 @@ TS20210903,,TS20211214,,CP05MOAS-GL339-01-ADCPAM000,14,1,2021-09-04T00:31:00,202
...
\ No newline at end of file
+TS20210903,,TS20211214,,CP05MOAS-GL339-05-PARADM000,14,1,2021-09-04T00:31:00,2021-12-15T05:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.06277,-70.35737,,200,200,EB,,
+AR66,,,,CP05MOAS-GL339-00-ENG000000,15,1,2022-04-09T21:16:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.08297,-70.367,,200,200,EB,,
+AR66,,,,CP05MOAS-GL339-01-ADCPAM000,15,1,2022-04-09T21:16:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-50865,40.08297,-70.367,,200,200,EB,,
+AR66,,,,CP05MOAS-GL339-02-FLORTM000,15,1,2022-04-09T21:16:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04472,40.08297,-70.367,,200,200,EB,,
+AR66,,,,CP05MOAS-GL339-03-CTDGVM000,15,1,2022-04-09T21:16:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09338,40.08297,-70.367,,200,200,EB,,
+AR66,,,,CP05MOAS-GL339-04-DOSTAM000,15,1,2022-04-09T21:16:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00604,40.08297,-70.367,,200,200,EB,,
+AR66,,,,CP05MOAS-GL339-05-PARADM000,15,1,2022-04-09T21:16:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.08297,-70.367,,200,200,EB,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL339_Deploy.csv:91]: looks good

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 2]
```diff
@@ -64,4 +64,10 @@ TS20211214,,TS20220221,,CP05MOAS-GL380-01-ADCPAM000,11,1,2021-12-15T07:36:00,202
...
\ No newline at end of file
+TS20211214,,TS20220221,,CP05MOAS-GL380-05-PARADM000,11,1,2021-12-15T07:36:00,2022-02-22T08:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50244,39.99217,-70.30902,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-00-ENG000000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.93425,-70.58905,,1000,1000,FZ-2,,
+AR66,,,,CP05MOAS-GL380-01-ADCPAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-50867,39.93425,-70.58905,,1000,1000,FZ-2,,
+AR66,,,,CP05MOAS-GL380-02-FLORTM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04251,39.93425,-70.58905,,1000,1000,FZ-2,,
+AR66,,,,CP05MOAS-GL380-03-CTDGVM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09037,39.93425,-70.58905,,1000,1000,FZ-2,,
+AR66,,,,CP05MOAS-GL380-04-DOSTAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00171,39.93425,-70.58905,,1000,1000,FZ-2,,
+AR66,,,,CP05MOAS-GL380-05-PARADM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50152,39.93425,-70.58905,,1000,1000,FZ-2,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL380_Deploy.csv:73]: looks good

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 3]
```diff
@@ -70,4 +70,16 @@ TS20220104,,TS20220221,,CP05MOAS-GL388-01-ADCPAM000,12,1,2022-01-04T20:57:00,202
...
+AR66,,AR66,,CP05MOAS-GL388-05-PARADM000,13,1,2022-04-10T16:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.93303,-70.58503,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL388-00-ENG000000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.9338,-70.66867,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL388-01-ADCPAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-43106,39.9338,-70.66867,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL388-02-FLORTM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-04467,39.9338,-70.66867,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL388-03-CTDGVM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09363,39.9338,-70.66867,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL388-04-DOSTAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00418,39.9338,-70.66867,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL388-05-PARADM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.9338,-70.66867,,1000,1000,FZ-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL388_Deploy.csv:85]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 4]
```diff
@@ -83,9 +83,9 @@ TS20210903,,TS20211214,,CP05MOAS-GL376-02-FLORTM000,14,1,2021-09-03T22:55:00,202
...
\ No newline at end of file
+TS20220104,,AR66,,CP05MOAS-GL376-00-ENG000000,15,1,2022-01-04T20:51:00,2022-04-15T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.94245,-70.38275,,1000,1000,SS-1,,
+TS20220104,,AR66,,CP05MOAS-GL376-01-ADCPAM000,15,1,2022-01-04T20:51:00,2022-04-15T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.94245,-70.38275,,1000,1000,SS-1,,
+TS20220104,,AR66,,CP05MOAS-GL376-02-FLORTM000,15,1,2022-01-04T20:51:00,2022-04-15T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03190,39.94245,-70.38275,,1000,1000,SS-1,,
+TS20220104,,AR66,,CP05MOAS-GL376-03-CTDGVM000,15,1,2022-01-04T20:51:00,2022-04-15T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09083,39.94245,-70.38275,,1000,1000,SS-1,,
+TS20220104,,AR66,,CP05MOAS-GL376-04-DOSTAM000,15,1,2022-01-04T20:51:00,2022-04-15T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00499,39.94245,-70.38275,,1000,1000,SS-1; top mount,,
+TS20220104,,AR66,,CP05MOAS-GL376-05-PARADM000,15,1,2022-01-04T20:51:00,2022-04-15T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50203,39.94245,-70.38275,,1000,1000,SS-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL376_Deploy.csv:91]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/CP05MOAS-GL339_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL339_Deploy.csv:91]: Looks good.

File: deployment/CP05MOAS-GL376_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL376_Deploy.csv:91]: Looks good.

File: deployment/CP05MOAS-GL380_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL380_Deploy.csv:73]: Looks good.

File: deployment/CP05MOAS-GL388_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL388_Deploy.csv:85]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**